test/memory_coloring_test.cpp

```diff
@@ -729,7 +729,7 @@ TEST_CASE(test39)
         run_pass(*smod);
     }
 
-    CHECK(mm->get_parameter_shape("scratch").bytes() == 4);
+    CHECK(mm->get_parameter_shape("scratch").bytes() == 1);
```

Comment:
Guessing this was changed because of the change to alignment then?

Reply:
yes